src/main/java/org/fundacionjala/coding/jimmy/encryption/VigenereCrypto.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.coding.jimmy.encryption;
+
+/**
+ * Class for overriding the methods.
+ */
+public class VigenereCrypto extends Crypto {
+
+    /**
+     * Vigenere encode override.
+     *
+     * @param objects are the parameters.
+     * @return the code cipher.
+     */
+    @Override
+    public String encode(Object... objects) {
+        if (isEmptyOrNull(objects[0])) {
+            return "";
+        }
+        return coreVigenere(objects[0].toString(), objects[1].toString(), Integer.parseInt(objects[1].toString()));
+    }
+
+    /**
+     * Vigenere decode override.
+     *
+     * @param objects are the parameters.
+     * @return the code cipher.
+     */
+    @Override
+    public String decode(Object... objects) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "encode" on line 15. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)